### DISCUSSION TITLE
Is there a debug argument for command line?

### DISCUSSION THREAD

**Discussion by mozthefox** [Stability-AI/StableSwarmUI#262] [Q&A] [answered]
Hey!

I'm using Swarm on the Mac and I wondered if there's a switch/argument I can add to the launch command to enable debug output? I find live view of generation progression really useful in the command line when using AUTOMATIC1111 and wondered if the same feature was available in Swarm?

(I know there is a live log view built in, but this means I need an additional browser window open to view it - I regularly have 30+ tabs open so I'd rather view live progress in the CLI. Also the debug info has more valuable information than the progress bars in Swarm's Generate tab :) )

Many thanks,
James

**Comment by mcmonkey4eva** [comment, marked as answer]
You can set the global log level under server settings which will persist, or if you love CLIs you can of course just do `--loglevel debug` https://github.com/Stability-AI/StableSwarmUI/blob/master/docs/Command%20Line%20Arguments.md

**Reply by mozthefox** [reply]
Ahh, I'm so sorry! I must have checked the docs 5 times and completely missed that page. Thank you so much!
